src/overlays/actors/ovl_En_Horse/z_en_horse.c

```diff
@@ -2850,10 +2753,10 @@ void EnHorse_BridgeJumpInit(EnHorse* this, GlobalContext* globalCtx) {
 void EnHorse_StartBridgeJump(EnHorse* this, GlobalContext* globalCtx) {
     this->postDrawFunc = EnHorse_BridgeJumpInit;
     if (this->bridgeJumpIdx == 0) {
-        globalCtx->csCtx.segment = SEGMENTED_TO_VIRTUAL(D_02002AC0);
+        globalCtx->csCtx.segment = SEGMENTED_TO_VIRTUAL(spot09_sceneCutsceneData0x002AC0);
```

Comment:
This cutscene reference should use a proper name instead of using the name auto-generated by ZAP